### DISCUSSION TITLE
[Bug]: label available on portal https://plus.frigate.video/

### DISCUSSION THREAD

**Discussion by pilot1981** [blakeblackshear/frigate#13502] [Report a Bug]
### Checklist

- [X] I have updated to the latest available Frigate version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [incognito mode](https://www.computerworld.com/article/1719851/how-to-go-incognito-in-chrome-firefox-safari-and-edge.html) to rule out problems with any third party extensions or plugins I have installed.

### Describe the problem you are having

Using portal https://plus.frigate.video/, if I select only license_plate label and I go to verify each single image, I have in GUI view only person label visible...I don't this this is a aesthetics problem.

So after I marked many images with license_plate model, I gone to enable also person model, I can back to images and I see they are all marked with person model...

### Steps to reproduce

1. I select only license_plate label
2. I go to verify each single image 
3. I marked it as license_plate
4.  I have in GUI view only person label visible.

### Version

lastest

### Frigate config file

```yaml
NA
```

### docker-compose file or Docker CLI command

```yaml
NA
```

### Relevant Frigate log output

```shell
NA
```

### Relevant go2rtc log output

```shell
NA
```

### Operating system

HassOS

### Install method

HassOS Addon

### Network connection

Wired

### Camera make and model

NA

### Screenshots of the Frigate UI's System metrics pages

NA

**Comment by NickM-27** [comment]
This is the repo for Frigate itself. Email bugs to blake@frigate.video
